opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -232,9 +232,9 @@ message Profile {
   // interpretation of results.
 
   // Time of collection (UTC) represented as nanoseconds past the epoch.
-  int64 time_nanos = 4;
+  uint64 time_nanos = 4;
   // Duration of the profile, if a duration makes sense.
-  int64 duration_nanos = 5;
+  uint64 duration_nanos = 5;
```

Comment:
Should you use fixed64? Since we are using anyway a large part of the bytes (Time.now()), I think is more efficient.

Reply:
I also renamed to `_unix_nano` to be consistent with the other signals.